### PR TITLE
Added details to issue template

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -14,13 +17,23 @@ Version Information:
...
+
+1. {{add step 1}}
+2. {{add step 2}}
+3. {{add step 3}}
 -->
 
 
 <!--
-Some guidelines:
+Before you report this issue, please consider the following:
```
Comment (czardoz) [inline .github/ISSUE_TEMPLATE.md]: Too verbose, and asking users to "consider" doing way too many things.
How about just adding a fourth point, "For security issues, please contact us at security@getpostman.com, you will receive a response within 24 hours."

File: test/integration/tc4/tc4.postman_collection.json [hunk 2]
```diff
@@ -1,40 +1,40 @@
 {
-  "variables": [],
```
Comment (czardoz) [inline test/integration/tc4/tc4.postman_collection.json]: Why is this in a PR titled "Added details to issue template"?

Comment (kunagpal) [inline test/integration/tc4/tc4.postman_collection.json]: This fixes failing builds due to a breaking changes in Postman Echo.

Comment (czardoz) [inline test/integration/tc4/tc4.postman_collection.json]: Alrighty :+1:

**Pull request merged**